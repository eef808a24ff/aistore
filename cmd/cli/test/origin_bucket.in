ais create bucket $BUCKET

ais set props ais://$BUCKET backend_bck=ais://something // FAIL "backend bucket should point to cloud bucket"
ais set props ais://$BUCKET backend_bck=cloud://something // FAIL "backend bucket should point to cloud bucket"
ais set props ais://$BUCKET backend_bck=gcp:// // FAIL "backend bucket name should not be empty"

ais show props ais://$BUCKET backend_bck -v
