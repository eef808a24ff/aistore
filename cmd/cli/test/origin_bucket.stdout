"$BUCKET" bucket created
 PROPERTY               VALUE
backend_bck.name
backend_bck.provider
